requirements.txt

```diff
@@ -1,3 +1,4 @@
+argcomplete [ARGCOMPLETE]
```

Comment:
Where is this syntax documented? How do I use it? `requirements.txt` is only for setting up the development environment; `setup.py` is where the actual dependencies go.

Reply:
It's documents as [part of PEP 508](https://www.python.org/dev/peps/pep-0508/#extras), but I originally misread the usage.  I'll remove the `[ARGCOMPLETE]` tag.

Reply:
Addressed by TimothySprague/scuba@ea1be7f6fb2a25a586329f831373ff28e7a0a2db